OneCommunityGlobal/HighestGoodNetworkApp
# Harshida - Change button color and alignment of Timer component.

# Description
This PR for the button color of timer on dashboard. Changed the blue color. Also when resize the browser, buttons (Start / Pause / Stop ) overlap with clear and timer. so fixed the alignment as well.  

## Related PRS (if any):
This PR is not related with any PR.

## Main changes explained:

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. logged In as any user 
4. go to dashboard
5. Verify the below changes on timer:
      - Click on the Start button. Verify the color of Pause button (Blue) and mouse hover color(Just dark blue)
      - Now resize the browser. Verify the buttons alignments.
      
## Screenshots or videos of changes:
Before Changes: 
<img width="349" alt="Before_Changes" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/109878936/6f2b21f1-95ea-4cf8-9a3e-96ef60d2f2eb">

After Changes:
<img width="376" alt="After_Changes" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/109878936/6cf17c7a-f815-43be-9745-950e990f8552">


## Note:
